.changeset/curly-otters-guess.md

```diff
@@ -0,0 +1,5 @@
+---
+'vercel': minor
```

Comment:
Question: Should this be "major"?

Reply:
The 2 possible "breaking" changes in this PR:
- `vc logs` doesn't return build logs anymore
- `vc inspect` returns new exit code (3) when deployment is not ready

Reply:
`vc logs` still returns build logs in this PR.

Reply:
Team decision: we're not changing the exit codes yet, I'll revert this change